x/programs/runtime/import_program.go

```diff
@@ -17,10 +17,10 @@ const (
 )
 
 type callProgramInput struct {
-	ProgramID    ids.ID
-	FunctionName string
-	Params       []byte
-	Fuel         uint64
+	ProgramID ids.ID
+	Function  []byte
```

Comment:
Why was this changed?

Reply:
Yeah, I think it makes more sense as a string.

Reply:
This was unintended, Rolling back!